### PR TITLE
Merge pull request #25 from pixelize/focus-styles

### PR DESCRIPTION
**Pull request by pixelize**
Hi @dsznajder 
I've added a new prop: `focusStyles`
It allows the developper to add some custom styles on the Input when its focused.
(i.e: change the backgroundColor, apply transform, change font size or what ever you want ;) ...).

Hope it fits your project and you will merge it asap, happy to contribute :)
Also, I can update the documentation when it will be up !

Demo use case: 

ezgif-3-c9ca542a7680

### CODE REVIEW

File: src/OtpInput.tsx [hunk 1]
```diff
@@ -63,11 +65,14 @@ export default class OtpInput extends Component<Props, State> {
       value,
     } = this.props
 
+    console.log(focusStyles)
```
Comment (dsznajder) [inline src/OtpInput.tsx]: Remove logs :)

Comment (pixelize) [reply src/OtpInput.tsx]: oops, my bad, i thought i discarded this one ;) Done !

**Comment by dsznajder** [comment]
Thanks :) I will release it today :)

**Comment by pixelize** [comment]
Amazing, thanks for the quick reaction, so we will be able update our project dependency ;)

**Comment by dsznajder** [comment]
3.1.0 released :)

**Comment by 3braintechnologies** [comment]
Focus style does not work when you used onFocus and onBlur props.

**Pull request merged**